tool/lib/iso15765_2.py

```diff
@@ -37,10 +38,7 @@ class IsoTp:
     def __init__(self, arb_id_request, arb_id_response, bus=None):
         # Setting default bus to None rather than the actual bus prevents a CanError when
         # called with a virtual CAN bus, while the OS is lacking a working CAN interface
-        if bus is None:
```

Comment:
Is this logic handled elsewhere now, to prevent `CanError` when called with a virtual CAN bus while the OS is lacking a working CAN interface?